src/pages/chanyoung/Main/components/feeds/Comment.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import CommentWnd from './CommentWnd';
+import CommentForm from './CommentForm';
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentsInfo: {
+        list: [],
+        nextKey: null,
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.initComment();
+  }
+
+  initComment = () => {
+    fetch('http://localhost:3000/chanyoung/data/commentData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState(() => ({
+          commentsInfo: {
+            list: data,
+            nextKey: data.length + 1,
+          },
+        }));
+      });
+  };
+
+  addCommnet = e => {
+    e.preventDefault();
+    const { nextKey } = this.state.commentsInfo;
+    const newComment = {
+      key: nextKey,
+      name: '정찬영',
+      comment: e.target[0].value,
+    };
+
+    this.setState(({ commentsInfo: { list } }) => {
+      return {
+        commentsInfo: {
+          list: [...list, newComment],
+          nextKey: nextKey + 1,
+        },
+      };
+    });
```

Comment:
```suggestion
    this.setState(prevState => {
      const { list } = prevState.commentsInfo
      return {
        commentsInfo: {
          list: [...list, newComment],
          nextKey: nextKey + 1,
        },
      };
    });
```
구조분해 할당 Depth가 너무 깊어져서 알아보기 어려운거 같아요